Kickstarter-iOS/Views/Cells/ProjectPamphletCreatorHeaderCell.swift

```diff
@@ -0,0 +1,134 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+private enum Layout {
+  enum Button {
+    static let height: CGFloat = 48
+    static let width: CGFloat = 152
+  }
+}
+
+protocol ProjectPamphletCreatorHeaderCellDelegate: class {
+  func projectPamphletCreatorHeaderCellDidTapButton(
+    _ cell: ProjectPamphletCreatorHeaderCell,
+    project: Project
+  )
+}
+
+final class ProjectPamphletCreatorHeaderCell: UITableViewCell, ValueCell {
+  // MARK: Properties
+
+  private let launchDateLabel: UILabel = { UILabel(frame: .zero) }()
+  private let rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private let viewProgressButton: UIButton = { UIButton(frame: .zero) }()
+  private let viewModel: ProjectPamphletCreatorHeaderCellViewModelType =
+    ProjectPamphletCreatorHeaderCellViewModel()
+
+  internal weak var delegate: ProjectPamphletCreatorHeaderCellDelegate?
+
+  // MARK: Lifecycle
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+    self.configureViews()
+    self.setupConstraints()
+    self.bindViewModel()
+  }
+
+  required init?(coder _: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: Configuration
+
+  internal func configureWith(value project: Project) {
+    self.viewModel.inputs.configure(with: project)
+  }
+
+  private func configureViews() {
+    _ = ([self.launchDateLabel, self.viewProgressButton], self.rootStackView)
+      |> ksr_addArrangedSubviewsToStackView()
+
+    _ = (self.rootStackView, self.contentView)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToMarginsInParent()
+
+    self.viewProgressButton.addTarget(
+      self, action: #selector(self.viewProgressButtonTapped), for: .touchUpInside
+    )
+  }
+
+  private func setupConstraints() {
+    NSLayoutConstraint.activate([
+      self.viewProgressButton.heightAnchor.constraint(equalToConstant: Layout.Button.height),
+      self.viewProgressButton.widthAnchor.constraint(equalToConstant: Layout.Button.width)
+    ])
+  }
+
+  // MARK: - View model
+
+  override func bindViewModel() {
+    super.bindViewModel()
+    self.launchDateLabel.rac.attributedText = self.viewModel.outputs.launchDateLabelAttributedText
+    self.viewProgressButton.rac.title = self.viewModel.outputs.buttonTitle
+
+    self.viewModel.outputs.notifyDelegateViewProgressButtonTapped
+      .observeForUI()
+      .observeValues { [weak self] in
```

Comment:
Can we name the argument here? We typically name them when observing the values in `bindViewModel` to make it clear what the value is.